src/components/ResponsiveAppBar.jsx

```diff
@@ -206,7 +204,7 @@ const ResponsiveAppBar = () => {
                 if (page.url) {
                   return (
                     <MenuItem
-                      key={`Mobile-${page.translationKey}`}
+                      key={page.translationKey}
```

Comment:
Why do you remove those key prefixes?

Reply:
Extremely sorry. I guess I did it mistakenly. I will update it right now. Sorry.

Reply:
No problem, I was just wondering if they  were a specific reason I forget to do it